cubical/machines/interval_gain_machine.py

```diff
@@ -196,10 +196,12 @@ def determine_allocators(cls, options):
 
     @classmethod
     def get_full_kernel(cls, options, diag_gains):
+
```

Comment:
Again, please minimise the diff by removing added whitespace and testing artifacts.